pom.xml

```diff
@@ -309,19 +309,19 @@
         <version.org.apache.ws.security>2.2.3</version.org.apache.ws.security>
         <version.org.apache.ws.xmlschema>2.2.4</version.org.apache.ws.xmlschema>
         <version.org.apache.xalan>2.7.1.jbossorg-4</version.org.apache.xalan>
-        <version.org.bouncycastle>1.60</version.org.bouncycastle>
+        <version.org.bouncycastle>1.62</version.org.bouncycastle>
```

Comment:
JBossWS/CXF is still using 1.60. Let's upgrade this version in CXF first to see if all work, then we back to upgrade WFLY.